### PR TITLE
Migration of libraryBrowser and multiDownload to ES6 modules

### PR DESCRIPTION
**Pull request by Camc314**
**Issues**
part of #868

### CODE REVIEW

**Comment by dmitrylyzo** [comment]
I think for temporary backward compatibility we should use
```js
libraryBrowser = libraryBrowser.default || libraryBrowser;
```
In this case we will just remove them when module becomes ES6

File: src/scripts/libraryBrowser.js [hunk 1]
```diff
@@ -1,199 +1,212 @@
...
-            return query;
-        },
-        saveQueryValues: function (key, query) {
-            var values = {};
-
-            if (query.SortBy) {
-                values.SortBy = query.SortBy;
-            }
+        if (options.addLayoutButton) {
+            html += '<button is="paper-icon-button-light" title="' + globalize.translate('ButtonSelectView') + '" class="btnChangeLayout autoSize" data-layouts="' + (options.layouts || '') + '" onclick="LibraryBrowser.showLayoutMenu(this, \'' + (options.currentLayout || '') + '\');"><span class="material-icons view_comfy"></span></button>';
```
Comment (dmitrylyzo) [inline src/scripts/libraryBrowser.js:104]: What about `LibraryBrowser.showLayoutMenu`?
I think it is better to extract onclick as listener.

Comment (dmitrylyzo) [reply src/scripts/libraryBrowser.js:104]: Looks like it is impossible - this function just returns HTML.

File: src/scripts/libraryBrowser.js [hunk 2]
```diff
@@ -1,199 +1,212 @@
...
+export default {
+    getSavedQueryKey,
+    loadSavedQueryValues,
+    saveQueryValues,
+    saveViewSetting,
+    getSavedView,
+    showLayoutMenu,
+    getQueryPagingHtml,
+    showSortMenu
+};
```
Comment (dmitrylyzo) [inline src/scripts/libraryBrowser.js:212]:
```suggestion
const libraryBrowser = {
    getSavedQueryKey,
    loadSavedQueryValues,
    saveQueryValues,
    saveViewSetting,
    getSavedView,
    showLayoutMenu,
    getQueryPagingHtml,
    showSortMenu
};

window.LibraryBrowser = libraryBrowser;

export default libraryBrowser;
```

**Comment by Camc314** [comment]
> One missed
> https://github.com/jellyfin/jellyfin-web/blob/40b98bb3d49e377658efca5bd85c9c1500ca1bbd/src/controllers/movies/movies.js#L1-L4
> 
> 
> But it works without hack - returned object contains exported functions + `default`.

I'll add it anyway

**Pull request merged**